src/generator/www/index.html

```diff
@@ -20,10 +20,10 @@
     	<input type="url" class="form-control" id="url" name="url">
  		</br>
   		<label for="theme">Choose your theme</label>
-  		<select class="form-control" id="theme" name="theme">
-    		<option value="1">Light</option>
-    		<option value="2">Dark</option>
-  		</select>
+  		  <select class="form-control" id="theme" name="theme">
+    		  <option value="1">Light</option>
```

Comment:
values should be changed to light and dark. there is no change from what already exists in master now.